src/scanner.c

```diff
@@ -292,20 +295,46 @@ bool try_lex_script_start(Scanner *scanner, TSLexer *lexer)
   }
   skip_space_tabs(lexer);
 
+  // Skip :let-heredoc parameters (trim, eval, etc.)
```

Comment:
Wo boy, maybe we want to capture those at some point ? At least leave it as a TODO

Reply:
I ended up marking it as a TODO to avoid giving extra work to both of us; we could fix this in another PR.